pkg/logging/routing-integration_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2016-2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package logging_test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/aws/aws-node-termination-handler/pkg/logging"
+	h "github.com/aws/aws-node-termination-handler/pkg/test"
+
+	"github.com/rs/zerolog/log"
+)
+
+func TestIntegration_zerologInfo(t *testing.T) {
+	buf := &strings.Builder{}
+	errBuf := &strings.Builder{}
+
+	l := log.Output(logging.RoutingLevelWriter{Writer: buf, ErrWriter: errBuf})
+
+	const s = "this is a test"
+	l.Info().Msg(s)
```

Comment:
I like that you wrote an "integration" test suite as well, to demonstrate how logging actually occurs in this project. Nice work!